messaging-activemq/src/main/resources/schema/wildfly-messaging-activemq_3_0.xsd

```diff
@@ -0,0 +1,904 @@
+                        <xs:element name="param" minOccurs="0" maxOccurs="unbounded" type="paramType" />
+                    </xs:sequence>
+                    <xs:attribute name="name" type="xs:string" use="required" />
+                    <xs:attribute name="factory-class" type="xs:string" use="required" />
+                </xs:complexType>
+            </xs:element>
+
+            <xs:element name="http-acceptor" minOccurs="0" maxOccurs="unbounded">
+                <xs:complexType>
+                    <xs:sequence>
+                        <xs:element name="param" minOccurs="0" maxOccurs="unbounded" type="paramType" />
+                    </xs:sequence>
+                    <xs:attribute name="upgrade-legacy" type="xs:boolean" use="optional" />
+                    <xs:attribute name="name" type="xs:string" use="required" />
+                    <xs:attribute name="http-listener" type="xs:string" use="required" />
+                </xs:complexType>
+            </xs:element>
+
+            <xs:element name="remote-acceptor" minOccurs="0" maxOccurs="unbounded">
+                <xs:complexType>
+                    <xs:sequence>
+                        <xs:element name="param" minOccurs="0" maxOccurs="unbounded" type="paramType" />
+                    </xs:sequence>
+                    <xs:attribute name="name" type="xs:string" use="required" />
+                    <xs:attribute name="socket-binding" type="xs:string" use="required" />
+                </xs:complexType>
+            </xs:element>
+
+            <xs:element name="in-vm-acceptor" minOccurs="0" maxOccurs="unbounded">
+                <xs:complexType>
+                    <xs:sequence>
+                        <xs:element name="param" minOccurs="0" maxOccurs="unbounded" type="paramType" />
+                    </xs:sequence>
+                    <xs:attribute name="name" type="xs:string" use="required" />
+                    <xs:attribute name="server-id" type="xs:string" use="required" />
+                </xs:complexType>
+            </xs:element>
+
+            <xs:element name="acceptor" minOccurs="0" maxOccurs="unbounded">
+                <xs:complexType>
+                    <xs:sequence>
+                        <xs:element name="param" minOccurs="0" maxOccurs="unbounded" type="paramType" />
+                    </xs:sequence>
+                    <xs:attribute name="name" type="xs:string" use="required" />
+                    <xs:attribute name="factory-class" type="xs:string" use="required" />
+                </xs:complexType>
+            </xs:element>
+
+            <xs:element name="broadcast-group" minOccurs="0" maxOccurs="unbounded">
+                <xs:complexType>
+                    <xs:attribute name="name" type="xs:string" use="required" />
+
+                    <xs:attribute name="socket-binding" type="xs:string" use="optional" />
+                    <xs:attribute name="jgroups-stack" type="xs:string" use="optional" />
```

Comment:
could you add a deprecated annotation, please? 

```
<xs:annotation>
  <xs:documentation>
    Deprecated. Use jgroups-channel instead.
  </xs:documentation>
</xs:annotation>
```

Reply:
Will do.